### DISCUSSION TITLE
UNUSED and FOUND for same files...

### DISCUSSION THREAD

**Discussion by Nikoh77** [emmercm/igir#855] [Q&A]
Hello guys, I just discovered igir and I am trying to use it.
After some time I understod how it work but I need your help because on the report file i can see FOUND and UNUSED for same (few) files...
![Schermata a 2023-12-24 02-29-41](https://github.com/emmercm/igir/assets/7826178/52f5e563-faa6-4214-baad-9d4d84921ac6)
![Schermata a 2023-12-24 02-30-05](https://github.com/emmercm/igir/assets/7826178/4db94e10-1258-40fe-a867-0cec97da86cd)
Please help.

**Comment by emmercm** [comment]
This is definitely unexpected. Could you upload the entire CSV file, and the `igir` options used to generate it? Feel free to remove any identifying information from the CSV such as directory paths.
